benchmarks/linalg/QR_bm.cpp

```diff
@@ -0,0 +1,26 @@
+#include <benchmark/benchmark.h>
+#include <cytnx.hpp>
+
+// Cytnx test
+namespace BMTest_QR {
```

Comment:
The name of namespace should [be snake case](https://google.github.io/styleguide/cppguide.html#Namespace_Names). Maybe we can just put the test functions in `namespace cytnx {}` directly or in `namespace cytnx {namespace benchmark {}}`?